tmpl/_travis.yml

```diff
@@ -0,0 +1,33 @@
+language: node_js
+node_js: "6"
+os:
+- linux
+- osx
+dist: trusty
+osx_image: xcode8.3
+sudo: false
+
+cache:
+  directories:
+  - node_modules
+  - $HOME/.cache/electron
+
+addons:
+  apt:
+    packages:
+    - fakeroot
+    - rpm
+
+branches:
+  only:
+  - master
+  - /^v\d+\.\d+\.\d+/
```

Comment:
The only real changes from the demo versions I made the other day are:
* I added the "only branches" to Travis to mirror AppVeyor, and
* I fixed the version regex to accept numbers that are more than one digit.